src/Entity/User.php

```diff
@@ -7,12 +7,15 @@
 use Doctrine\Common\Collections\Collection;
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Component\Security\Core\User\UserInterface;
+use Rollerworks\Component\PasswordStrength\Validator\Constraints as RollerworksPassword;
 
 /**
  * @ORM\Entity(repositoryClass=UserRepository::class)
  */
 class User implements UserInterface
 {
+    public const ROLE_ADMIN = 'ROLE_ADMIN';
```

Comment:
What is this ?

Reply:
I think it can be useful if I want a admin user able to do special stuff like deleting a comment, deleting another user...

Reply:
I deleted this line